src/software/amazon/ion/ValueFactory.java

```diff
@@ -252,11 +252,6 @@
 
     /**
      * Constructs a new {@code list} with the given child.
-     * <p>
-     * <b>This method is temporary</b> until {@link #newList(Collection)} is
```

Comment:
I've added #209 to cover the investigation and possible removal of these methods, however, since `newList(Collection)` and `newSexp(Collection)`  do not exist anymore it isn't clear to me how to fix the javadoc without removing it.  What do you suggest would be an appropriate text?

Reply:
I'd suggest that the text itself is fine as is (although outdated), and that we just apply a minimal change here to make javadoc happy (e.g., change "{@link #newList(Collection)}" to "newList(Collection)"?).  Then #209 will address the more involved aspects here.